assets/app/styles/_core.less

```diff
@@ -112,9 +122,18 @@ html, body {
       float: left;
       width: @navbar-project-menu-sm;
     }
+    .navbar-filter-widget {
+      /* prevent the filter button from wrapping at wide
+         screen sizes */
+      min-width: 340px;
+    }
     .navbar-search {
       margin-left: @sidebar-left-width-open-sm;
     }
+    .create-button-container {
+      text-align: right;
+      margin: 30px 0;
```

Comment:
@ligit Right, I added the bottom padding so it doesn't sit right on top of the main container border when wrapped.

I get what you're saying about locality of reference, but I'm not sure how much it helps here. The other margins and sizes are specified elsewhere, so the context doesn't tell you too much. And I've added other styles like `text-align`. I'd rather be able to see the navbar styles all together in core.less.